contrib/zmq/zmq_sub.py

```diff
@@ -1,43 +1,70 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
 # Copyright (c) 2014-2016 The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+# A blocking example using python 2.7 can be obtained from the git history:
+# https://github.com/bitcoin/bitcoin/blob/37a7fe9e440b83e2364d5498931253937abe9294/contrib/zmq/zmq_sub.py
+
+import array
 import binascii
-import zmq
+import asyncio, zmq, zmq.asyncio
```

Comment:
nit: prefer imports to be on individual line (PEP8 E401), and listed individually.